### ISSUE TITLE
Publish addon in the new Add-on Marketplace

### ISSUE THREAD

**Issue by pacive** [boc-tothefuture/openhab-jruby#281]
Until the addon is properly integrated in openhab-addons I think it would be great to publish it in the new [Add-on Marketplace](https://community.openhab.org/c/marketplace/69) for easier installation.

**Comment by jimtng** [comment]
It appears that nothing special is required to post jruby binding on the add-on marketplace. We simply need to include an installation instructions + where to download the jar and how to set up the basic environment.

**Comment by jimtng** [comment]
I've just noticed this in OpenHAB Main UI -> Addons -> Automation - must be since 3.2M4?

It seems that things on the Add-on Marketplace can appear in OpenHAB MainUI and can be easily installed through it. I've just tried installing JRule by clicking Install, and it worked beautifully.

Of course since @boc-tothefuture is submitting the jruby binding to the official openhab-addons repository, and once that's accepted, the addon marketplace would be a moot point.

![image](https://user-images.githubusercontent.com/2554958/140743865-0821ebe6-6e33-48cc-a555-c4a106f193e4.png)
